deployment-examples/azure/serverless/notification/README.md

```diff
@@ -0,0 +1,15 @@
+## Deploy as Azure function app
+
+1. In order to setup the notification module config, follow our [how to run guide](../../../../notification/docs/HowToRun.md).
+2. Extension module requires `ADYEN_INTEGRATION_CONFIG` as `environment variable` or `.notificationrc` file to configure with [external file configuration](../../../../notification/docs/HowToRun.md#external-file-configuration).
```

Comment:
What's the reason for `.notificationrc`?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/993/commits/9a641f16cdbcf60dbd91701cc93d098faa79e0a1

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/993/commits/addec52897da5c24a72102e9b6438dfcd40ca2d4

Removed rc file usage in Azure and AWS docs.